RSDKv5/RSDK/Input/Wii/WiiInputDevice.cpp

```diff
@@ -77,10 +73,8 @@ void RSDK::SKU::InputDeviceWii::UpdateInput() {
         this->stateZ      |= (this->buttonMasksGC & 0) != 0;
         this->stateStart  |= (this->buttonMasksGC & PAD_BUTTON_START) != 0;
         this->stateSelect |= (this->buttonMasksGC & PAD_TRIGGER_Z) != 0;
-        this->stateUp     |= (PAD_StickY(0) > 10) ? 1 : 0;
-        this->stateDown   |= (PAD_StickY(0) < -10) ? 1 : 0;
-        this->stateLeft   |= (PAD_StickX(0) < -10) ? 1 : 0;
-        this->stateRight  |= (PAD_StickX(0) > 10) ? 1 : 0;
+        this->vDelta_L = (float)PAD_StickY(0) / (101 / 2);
+        this->hDelta_L = (float)PAD_StickX(0) / (101 / 2);
```

Comment:
Same.
Also where does 101 come from? Consider writing it as a `#define` to have an explanation of what this value is.

Reply:
101 is the maximum value for the left analog stick on a GC controller, and I couldn't find an existing variable for it in libOGC.